src/dataset-builder/DatasetBuilder.ts

```diff
@@ -324,7 +309,7 @@ export const CohortSelector = ({
       objectSets: [
         {
           values: cohorts,
-          header: 'Saved cohorts',
+          header: selectedCohorts?.length > 0 ? selectedCohorts[0].header : '',
```

Comment:
I'm wondering about this part of the change. Does this display need a header at all? What is it telling the user? I think it needs a prompt when the list is empty, but I don't think it needs a header if there's a cohort present.

Reply:
I don't think this needs a header in order to display it. But, the selector for this object set is used in other spots where it does need a header. And I think we use the header to group the cohorts (So, I don't think it matters what it is, as long as it is the same across cohorts). 
I think this would be a bigger refactor to move away from setting the header, but I'm new to this codebase so maybe it is not!